config/vufind/searches.ini

```diff
@@ -157,10 +157,18 @@ always_display_reset_filters = false
 ; commented out, fallback ID lookups will be disabled.
 ;fallback_id_field = previous_id_str_mv
 
+; This controls whether results are loaded with JavaScript when paging or changing
+; settings. Loading results this way improves performance and accessibility, and is
+; enabled by default. Loading will automatically fall back to non-JS mode if
+; JavaScript is not supported by the browser.
+load_results_with_js = true
```

Comment:
Since this setting is implemented in the base class, should we copy this setting to other backend-specific .ini files (Summon, EDS, etc., etc.)?

Reply:
Done.